lib/tictactoe.rb

```diff
@@ -20,6 +20,16 @@ def initialize (game_io, player1, player2)
   end
 
   def play_game
+    begin
     @game_loop.game_loop
+    @board.new_game
```

Comment:
Instead of mutating the board, would it make sense to throw away and spin up a new one?

Generally, if you're "starting again" or "repeating a process", things get simpler to understand and debug if you just dump everything you don't need anymore and start again. In the long term, this prevents lots of headaches.